enarx-keep-sgx-shim/src/start.S

```diff
@@ -154,15 +187,42 @@ enclave:
     .globl syscall
     .type syscall, @function
 syscall:
-    savep                                           # Save preserved registers
-    mov     %rsp,                   SRSP(%rcx)      # Save restoration stack pointer
-
-    mov     0x38(%rsp),             %r10            # Export %r10
-    mov     0x40(%rsp),             %r11            # Export %rax in %r11
-    mov     0x48(%rsp),             %rsp            # Get the exit context
-
-    xor     %rcx,                   %rcx            # Clear %rcx
-    zerof   %rcx                                    # Clear CPU flags
+    # Save preserved registers and restoration
+    # stack pointer
+    savep
+    mov     %rsp,                   SRSP(%rcx)
+
+    # Export %r10, %rax in %r11, and get exit
+    # context
+    mov     0x38(%rsp),             %r10
+    mov     0x40(%rsp),             %r11
+    mov     0x48(%rsp),             %rsp
+
+    # Clear %rcx and CPU flags
+    xor     %rcx,                   %rcx
+    zerof   %rcx
```

Comment:
Please don't change comments unnecessarily. Assembly is hard enough to get right. We don't need to add patch churn to the problem.

Reply:
For the reference this is `xcr0`, no `cr0`

Reply:
Sure, I can put it back. I was trying to make it more readable with the huge chunk zeroing the xsave header.